### PR TITLE
new package: enblend

### LINKED ISSUE #1157: Package request: enblend/enfuse
anyone tried to install enblend/enfuse http://enblend.sourceforge.net/

### PR DESCRIPTION
**Pull request by xtkoba**
Closes #1157.

### CODE REVIEW

File: packages/enblend/build.sh [hunk 1]
```diff
@@ -0,0 +1,15 @@
+TERMUX_PKG_HOMEPAGE=http://enblend.sourceforge.net/
+TERMUX_PKG_DESCRIPTION="A tool for compositing images using a Burt&Adelson multiresolution spline"
+TERMUX_PKG_LICENSE="GPL-2.0"
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_VERSION=4.2.0_p20161007
```
Comment (xeffyr) [inline packages/enblend/build.sh:5]: `_` is not allowed in version field.

**Review by xeffyr — CHANGES_REQUESTED** [review]

**Review by kornbik — COMMENTED** [review]
#8354

File: packages/enblend/build.sh (hunk 1 above)
Comment (xtkoba) [reply packages/enblend/build.sh]: Fixed. Thanks!

**Review by kornbik — COMMENTED** [review]
#8354

**Review by xeffyr — APPROVED** [review]

**Pull request merged**